test/hotspot/jtreg/compiler/loopopts/superword/TestMemorySegment.java

```diff
@@ -671,13 +664,14 @@ static Object[] testLongLoop_longIndex_intInvar_byte(MemorySegment a, int invar)
     }
 
     @Test
-    @IR(counts = {IRNode.LOAD_VECTOR_B, "= 0",
-                  IRNode.ADD_VB,        "= 0",
-                  IRNode.STORE_VECTOR,  "= 0"},
-        applyIfPlatform = {"64-bit", "true"},
-        applyIfCPUFeatureOr = {"sse4.1", "true", "asimd", "true"})
+    // @IR(counts = {IRNode.LOAD_VECTOR_B, "= 0",
+    //               IRNode.ADD_VB,        "= 0",
+    //               IRNode.STORE_VECTOR,  "= 0"},
+    //     applyIfPlatform = {"64-bit", "true"},
+    //     applyIfCPUFeatureOr = {"sse4.1", "true", "asimd", "true"})
     // FAILS: invariants are sorted differently, because of differently inserted Cast.
     // See: JDK-8330274
+    // Interestingly, it now passes for native, but not for objects.
```

Comment:
The same.  May be skip these 2 tests.

Reply:
See my suggestion.

Reply:
Removed the commented IR rule, will continue working on it with JDK-8330274